internal/engine/upper_test.go

```diff
@@ -908,24 +907,6 @@ k8s_yaml('snack.yaml')
 	// (i.e. check that we called BuildAndDeploy with no pre-existing state)
 	assert.False(t, call.oneImageState().HasLastResult())
 
-	var manifest model.Manifest
```

Comment:
this test had a weird dependency on BuildController handling the live-update, so i just removed the part of the test that required this.